Cargo.toml

```diff
@@ -44,6 +44,8 @@ wayland-csd-adwaita-crossfont = ["sctk-adwaita", "sctk-adwaita/crossfont"]
 wayland-csd-adwaita-notitle = ["sctk-adwaita"]
 android-native-activity = ["android-activity/native-activity"]
 android-game-activity = ["android-activity/game-activity"]
+kms = ["drm", "input", "xkbcommon", "udev", "parking_lot"]
+kms-ext = ["libseat"]
```

Comment:
Is it still necessary for split the feature into `kms` and `kms-ext` like this? I forget what the original rationale for this was.

Reply:
`kms-ext` uses (`libseat`)[https://sr.ht/~kennylevinsen/seatd/].

> Seat management takes care of mediating access to shared devices (graphics, input), without requiring the applications needing access to be root.

So this lib links/uses existing methods such as logind to create a user "seat" and allow access to various things. The lib is also a good intermediate step between winit and systemd-logind (plus others).

In our case this would allow anyone using this feature to run via TTY without requiring root. Standard `kms` here does require root access.

Unfortunately this feature is broken in this PR.

```
❯ cargo run --example window --no-default-features --features=kms-ext
   Compiling libseat-sys v0.1.7
   Compiling slog v2.7.0
   Compiling libseat v0.1.7
   Compiling winit v0.28.4 (/home/luke/Projects/winit-drm)
error: The platform you're compiling for is not supported by winit
  --> src/platform_impl/mod.rs:68:1
   |
68 | compile_error!("The platform you're compiling for is not supported by winit");
```